### ISSUE TITLE
Exp CLI: Check the alpha of the PNG before creating the IPA

### ISSUE THREAD

**Issue by lolobosse** [expo/expo#1248]
Hi there,

First 👏 for the work! It's smooth and cool 👍 

Second, I have found a annoying edge case which can be solved: when uploading a ios build, RGB**A** images are not tolerated. Meaning that you will get that at the end of the journey:

![image](https://user-images.githubusercontent.com/5024077/34898854-8eb93bbe-f7f5-11e7-9a9d-b61d5b6ec92f.png)

So what about checking it **BEFORE** creating the build to avoid an expensive process and/or writting **EXPLICITILY** in the docs?

Keep coding, you're awesome 😉

**Comment by terribleben** [comment]
Hey there! Did you check out our [App Icons](https://docs.expo.io/versions/latest/guides/app-icons.html) doc? There is an iOS instruction in there not to make your icon have any transparency. Lots of other useful tips as well.

**Comment by lolobosse** [comment]
@terribleben Hmpf, ok, feeling silly yet 🤣 . Still I'm convinced that it would be cool if CLI could check it (for lazy guys like me). I might give a try to update the CLI ;)

**Comment by terribleben** [comment]
You're right, we could add something in our CLI to check this for you. I think we can leave this issue open to track that!

**Comment by lolobosse** [comment]
👍

**Comment by lolobosse** [comment]
Actually, I might have an even better idea which would be to replace the transparency directly like [here](https://stackoverflow.com/questions/2322750/replace-transparency-in-png-images-with-white-background):

```
convert -background white -alpha remove ~/src.png ~/dest.png
```
No ⚠️ , no 💥 just smoooth 😉

**Comment by themakerman** [comment]
+1 :)
